traffic_ops/traffic_ops_golang/auth.go

```diff
@@ -58,3 +63,19 @@ func PrivLevel(privLevelStmt *sql.Stmt, user string) int {
 		return privLevel
 	}
 }
+
+func getPrivLevel(ctx context.Context) int {
```

Comment:
Consider making these helper functions return errors, rather than sentinel values, if the `Context` is missing the key. Using `Context`, you can't help it being a runtime error instead of a compile-time error, but you can still indicate when it's a code error, as opposed to a user request error, to make it easier to track down.

Reply:
Good call will return errors and let the caller decide how to handle it.